lambda/src/main/scala/pricemigrationengine/handlers/EstimationHandler.scala

```diff
@@ -80,18 +81,26 @@ object EstimationHandler extends CohortHandler {
   /*
    * Earliest start date spread out over 3 months.
    */
-  private[handlers] def spreadEarliestStartDate(
+  def spreadEarliestStartDate(
       subscription: ZuoraSubscription,
       invoicePreview: ZuoraInvoiceList,
       earliestStartDate: LocalDate
   ): IO[ConfigFailure, LocalDate] = {
 
+    def relu(number: Int): Int =
```

Comment:
the `.between` function will return a negative integer if the `customerAcceptanceDate.plusMonths(13)` is before the `earliestStartDate`. This will ensure those subscriptions do not get a start date earlier than the earliestStartDate.